lib/classes/PluginManager.js

```diff
@@ -90,6 +90,9 @@ class PluginManager {
 
         this.addPlugin(Plugin);
       } catch (error) {
+        if (this.cliCommands[0] === 'plugin') {
```

Comment:
For now I think this is ok (to generally ignore errors on the plugin plugin) to get it working.

On a longer term, we could introduce a boolean property `loadAlways` for plugins that we check here. So any plugin can request to be loaded, even if there are errors. But that approach is out of scope for this PR and would need some deeper thinking 😃